streams/pr.m3u

```diff
@@ -15,3 +15,5 @@ https://59514edd5dd8e.streamlock.net/unoradio/unoradio1/playlist.m3u8
 https://video1.getstreamhosting.com:1936/8212/8212/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="error",Telemundo WKAQ Noticias Puerto Rico (1080p) [Not 24/7]
 https://wkaqlive-lh.akamaihd.net/i/PR_STREAM1@311877/master.m3u8
+#EXTINF:-1 tvg-id="" status="",Wipr
+https://streamwipr.pr/hls/wipr/playlist.m3u8
```

Comment:
<img width="639" alt="image" src="https://user-images.githubusercontent.com/7253922/172072180-2b63cfdb-9833-4d4e-b921-30062070090d.png">

Reply:
it seems they've changed their address. going to address it.

Reply:
https://streannliveprtv.cachefly.net/prtv1/prtv1/playlist.m3u8